### PR TITLE
:seedling: add metrics for placement

### PR DESCRIPTION
**Pull request by haoqing0110**
## Related issue(s)

Ref: https://github.com/open-cluster-management-io/placement/pull/107

output
```
# HELP scheduling_bind_duration_seconds [ALPHA] How long in seconds it takes to bind a placement to placementDecisions.
# TYPE scheduling_bind_duration_seconds histogram
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1e-06"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-06"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-05"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.001"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.01"} 99
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.1"} 100
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1"} 100
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="10"} 100
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="100"} 100
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1000"} 100
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="+Inf"} 100
scheduling_bind_duration_seconds_sum{name="placement_scheduling"} 0.5276589120000001
scheduling_bind_duration_seconds_count{name="placement_scheduling"} 100
# HELP scheduling_plugin_duration_seconds [ALPHA] How long in seconds a plugin runs for a placement.
# TYPE scheduling_plugin_duration_seconds histogram
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="1e-06"} 0
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="9.999999999999999e-06"} 94
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="9.999999999999999e-05"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="0.001"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="0.01"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="0.1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="10"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="100"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="1000"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter",le="+Inf"} 100
scheduling_plugin_duration_seconds_sum{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter"} 0.0006524939999999999
scheduling_plugin_duration_seconds_count{name="placement_scheduling",plugin_name="Predicate",plugin_type="filter"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="1e-06"} 0
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="9.999999999999999e-06"} 94
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="9.999999999999999e-05"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="0.001"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="0.01"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="0.1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="10"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="100"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="1000"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer",le="+Inf"} 100
scheduling_plugin_duration_seconds_sum{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer"} 0.000540145
scheduling_plugin_duration_seconds_count{name="placement_scheduling",plugin_name="ResourceAllocatableMemory",plugin_type="prioritizer"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="1e-06"} 0
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="9.999999999999999e-06"} 0
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="9.999999999999999e-05"} 93
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="0.001"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="0.01"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="0.1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="1"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="10"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="100"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="1000"} 100
scheduling_plugin_duration_seconds_bucket{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter",le="+Inf"} 100
scheduling_plugin_duration_seconds_sum{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter"} 0.006523761
scheduling_plugin_duration_seconds_count{name="placement_scheduling",plugin_name="TaintToleration",plugin_type="filter"} 100
# HELP scheduling_scheduling_duration_seconds [ALPHA] How long in seconds it takes to schedule a placement.
# TYPE scheduling_scheduling_duration_seconds histogram
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="1e-06"} 0
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-06"} 0
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-05"} 13
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="0.001"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="0.01"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="0.1"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="1"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="10"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="100"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="1000"} 100
scheduling_scheduling_duration_seconds_bucket{name="placement_scheduling",le="+Inf"} 100
scheduling_scheduling_duration_seconds_sum{name="placement_scheduling"} 0.013472362999999998
scheduling_scheduling_duration_seconds_count{name="placement_scheduling"} 100
```

### CODE REVIEW

**Comment by haoqing0110** [comment]
/assign @qiujian16 @elgnay

plz help review the sample metrics and code, will add more testing code.

File: pkg/placement/plugins/steady/steady.go [hunk 1]
```diff
@@ -46,6 +49,13 @@ func (s *Steady) Description() string {
 
 func (s *Steady) Score(
 	ctx context.Context, placement *clusterapiv1beta1.Placement, clusters []*clusterapiv1.ManagedCluster) (plugins.PluginScoreResult, *framework.Status) {
+	startTime := time.Now()
```
Comment (qiujian16) [inline pkg/placement/plugins/steady/steady.go:52]: if this is the common thing, could we move this to the scheduler?

Comment (haoqing0110) [reply pkg/placement/plugins/steady/steady.go]: yes, moved to scheduler.

**Pull request merged**